xsl/pretext-html.xsl

```diff
@@ -1934,7 +1934,7 @@ along with MathBook XML.  If not, see <http://www.gnu.org/licenses/>.
     <xsl:value-of select="$b-commentary" />
 </xsl:template>
 <xsl:template match="fn|p|blockquote|biblio|biblio/note|interactive/instructions|gi|&DEFINITION-LIKE;|&OPENPROBLEM-LIKE;|&EXAMPLE-LIKE;|&PROJECT-LIKE;|task|&FIGURE-LIKE;|&THEOREM-LIKE;|&PROOF-LIKE;|case|&AXIOM-LIKE;|&REMARK-LIKE;|&COMPUTATION-LIKE;|&ASIDE-LIKE;|poem|assemblage|paragraphs|&GOAL-LIKE;|exercise|&SOLUTION-LIKE;|&DISCUSSION-LIKE;|exercisegroup|men|mrow|li[not(parent::var)]|contributor|fragment" mode="xref-as-knowl">
-    <xsl:value-of select="not($b-skip-knowls)" />
+    <xsl:value-of select="not($b-skip-knowls) and not($html-xref-render-style='link')" />
```

Comment:
Because of how the pub attributes work, you know that `$html-xref-render-style` will either be `'mixed'` xor `'link'`. So is a `not(...)` construct better than just using `$html-xref-render-style = 'mixed'`?

Reply:
The intention was that there might conceivably be some third option (or more) added. This construct guarantees that anything new would default to the "yes, generate knowls" path.

Perhaps such future-proofing should be left for the future? Trying to be as conservative as possible here.